### PR TITLE
gh-107910 Update not needing newline error message

### PR DESCRIPTION
**Pull request by xncbf**
I've looked everywhere I can, but there may be places I've missed.
Please feel free to comment if any corrections are needed

* Issue: gh-107910

### CODE REVIEW

File: Lib/urllib/parse.py [hunk 1]
```diff
@@ -253,62 +253,58 @@ def _hostinfo(self):
...
     'ParseResult', 'scheme netloc path params query fragment')
 
-_DefragResultBase.__doc__ = """
+_DefragResultBase.__doc__ = """\
 DefragResult(url, fragment)
 
 A 2-tuple that contains the url without fragment identifier and the fragment
-identifier as a separate argument.
-"""
+identifier as a separate argument."""
```
Comment (serhiy-storchaka) [inline Lib/urllib/parse.py]: This is not very meaningful change.

File: Lib/urllib/parse.py [hunk 2]
```diff
@@ -253,62 +253,58 @@ def _hostinfo(self):
...
-_DefragResultBase.fragment.__doc__ = """
-Fragment identifier separated from URL, that allows indirect identification of a
-secondary resource by reference to a primary resource and additional identifying
-information.
-"""
+_DefragResultBase.fragment.__doc__ = (
+    "Fragment identifier separated from URL, that allows indirect "
+    "identification of a secondary resource by reference to a primary "
+    "resource and additional identifying information."
+)
```
Comment (serhiy-storchaka) [inline Lib/urllib/parse.py]: There is not a bug. Docstring can be multiline.

File: Lib/test/test_statistics.py [hunk 3]
```diff
@@ -1,7 +1,5 @@
-x = """Test suite for statistics module, including helper NumericTestCase and
-approx_equal function.
-
-"""
+x = ("Test suite for statistics module, including helper NumericTestCase and "
+     "approx_equal function.")
```
Comment (serhiy-storchaka) [inline Lib/test/test_statistics.py]: Where is it used?

File: Lib/test/test_descr.py [hunk 4]
```diff
@@ -989,8 +989,8 @@ class EditableScrollablePane(ScrollablePane,EditablePane): pass
 
     def test_mro_disagreement(self):
         # Testing error messages for MRO disagreement...
-        mro_err_msg = """Cannot create a consistent method resolution
-order (MRO) for bases """
+        mro_err_msg = ("Cannot create a consistent method resolution "
+                       "order (MRO) for bases ")
```
Comment (serhiy-storchaka) [inline Lib/test/test_descr.py:993]: You changed test, but where is the code which is tested?

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
These changes look controversial and unneeded. If there is a useful change (perhaps in test_descr), it is not shown.

File: Lib/urllib/parse.py (hunk 1 above)
Comment (xncbf) [reply Lib/urllib/parse.py]: As per your comment, I reverted the unnecessary code.
Thank you for your review :)

File: Lib/urllib/parse.py (hunk 2 above)
Comment (xncbf) [reply Lib/urllib/parse.py]: As per your comment, I reverted the unnecessary code.
Thank you for your review :)

File: Lib/test/test_statistics.py (hunk 3 above)
Comment (xncbf) [reply Lib/test/test_statistics.py]: As per your comment, I reverted the unnecessary code.
Thank you for your review :)

File: Lib/test/test_descr.py (hunk 4 above)
Comment (xncbf) [reply Lib/test/test_descr.py:993]: Modified the test code.

**Comment by xncbf** [comment]
I have made the requested changes; please review again
@serhiy-storchaka

**Comment by xncbf** [comment]
I looked at other error messages, but couldn't find a similar case.

**Review by serhiy-storchaka — APPROVED** [review]
LGTM. Thank you for your contribution @xncbf.

**Pull request merged**